index2.html

```diff
@@ -70,10 +70,10 @@ <h1>Hey there. I’m Franck sefu.<br>
                 </ul>
                 
             </section>
```

Comment:
- [ ] The social media icons as shown below does not include facebook. Please include the other two.

Your design
![image](https://user-images.githubusercontent.com/84156714/212306065-f7b1b0da-b40a-49f2-9197-3c5fffaf8775.png)

Figma Design
![image](https://user-images.githubusercontent.com/84156714/212306160-62c9f3cb-8c01-4676-b45b-5a3e44575c77.png)